apps/pie-docs/src/patterns/haptic-feedback/guidance/guidance.md

```diff
@@ -0,0 +1,139 @@
+
+{% notification {
+  type: "warning",
+  message: "Please note that our default haptic feedback types **cannot be overridden or altered**."
+} %}
+
+---
+
+## Soft haptic feedback
+
+Soft haptic feedback can serve various purposes in order to enhance the overall user experience in a subtle and nuanced manner.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/haptic-feedback/soft-haptic-feedback.svg",
+  alt: "Vertical short orange bar intersected by a horizontal line, representing soft haptic feedback signal.",
+  width: "344px"
+} %}
+
+#### Where can I use soft haptic feedback?
+
+{% list {
+    type: listTypes.icon,
+    iconName: "check-circle",
+    items: [
+        "**Button presses:** Provide a gentle confirmation for various button presses to acknowledge user input without being intrusive.",
+        "**Toggle and state changes:** Convey changes in toggle switches or activation of different states in the UI with a moderately pronounced haptic sensation.",
+        "**Dynamic adjustments:** Provide users with tactile feedback when making dynamic adjustments, such as using sliders, to indicate significant changes.",
+        "**Non-critical notifications:** Use soft haptic feedback for non-critical notifications to gently alert users without causing disruption.",
+        "**Subtle transitions:** Enhance subtle transitions or animations between UI elements for a smooth and cohesive experience.",
+        "**Incremental adjustments:** Offer soft haptic feedback for incremental adjustments, like scrolling or volume control, to provide a sense of granularity."
+    ]
+} %}
+
+---
+
+## Medium haptic feedback
+
+A medium-intensity haptic feedback which can convey a sense of importance and grab the user's attention. A good example of this might be the haptic feedback that occurs when a user navigates to a new screen, commonly known as a "screen transition haptic", which is designed to provide a tactile confirmation of the navigation action.
+
+{% contentPageImage {
+  src:"../../../../assets/img/patterns/haptic-feedback/medium-haptic-feedback.svg",
+  alt: "Vertical long orange bar intersected by a horizontal line, indicating medium haptic feedback signal.",
+  width: "344px"
+} %}
+
+#### Where can I use medium haptic feedback?
+
+{% list {
+    type: listTypes.icon,
+    iconName: "check-circle",
+    items: [
+        "**Important actions:** Reinforce the significance of actions such as submitting a form, making a purchase, or saving changes with a medium level of haptic feedback.",
+        "**Gesture recognition:** Use medium haptic feedback to confirm the recognition of gestures, such as pinch-to-zoom or swipe gestures."
+    ]
+} %}
+
+---
+
+## Success haptic feedback
+
+A medium-intensity haptic feedback used to highlight a confirmation of an action. You can use it for the successful submission of a form or completion of a task, providing users with a reassuring response.
```

Comment:
Using "feedback" on its own like this feels a bit unnatural because it's usually an uncountable noun, and here it's being used as a countable noun ("a ... feedback").
```suggestion
A medium-intensity haptic response used to highlight the confirmation of an action. You can use it for the successful submission of a form or completion of a task, providing users with a reassuring response.
```